### PR TITLE
Mentor profile

### PR DESCRIPTION
**Pull request by shahpreetk**
Changes: 
Resolved error, icons changed, UI cleant

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -14,28 +14,52 @@ import { CookiesProvider } from "react-cookie";
...
+  };
+  constructor(props) {
+    super(props);
+    const { cookies } = props;
+    this.state = {
+        loggedin: cookies.get('loggedin') || cookies.set('loggedin', "abcd", '/'),
+        uid: cookies.get('uid') || cookies.set('uid', "1234", '/')
+    };
+}
+  // componentDidMount(loggedin,uid, cookies){
```
Comment (yashk2000) [inline frontend/src/App.js:32]: Should we remove this commented code entirely if we're not using it?

File: frontend/src/pages/MentorPage.jsx [hunk 2]
```diff
@@ -1,13 +1,9 @@
 import React, { Component } from 'react'
-import { Link } from 'react-router-dom';
-import { Form, Button, Row, Col, Container } from 'react-bootstrap';
+// import { Link } from 'react-router-dom';
```
Comment (yashk2000) [inline frontend/src/pages/MentorPage.jsx:2]: This comment as well.

File: frontend/src/App.js (hunk 1 above)
Comment (shahpreetk) [reply frontend/src/App.js:32]: oops forgot about that
I'll do it

**Comment by QEDK** [comment]
* Instead of "Get Started for Free", can you make it "Get started  →" 
* In place of the "Contact me" link, make it into a button with a chat bubble emoji?

File: frontend/src/pages/MentorPage.jsx (hunk 2 above)
Comment (shahpreetk) [reply frontend/src/pages/MentorPage.jsx]: done

**Pull request merged**